config/txloader/load/harvestcraft/lang/pl_PL.lang

```diff
@@ -0,0 +1,1136 @@
+# item.candleberryseedItem.name=Candleberry Seed
+# item.waxItem.name=Candleberry Wax
+
+item.hardenedleatherhelmItem.name=Czapka z Utwardzonej Skóry
+item.hardenedleatherchestItem.name=Tunika z Utwardzonej Skóry
+item.hardenedleatherleggingsItem.name=Spodnie z Utwardzonej Skóry
+item.hardenedleatherbootsItem.name=Buty z Utwardzonej Skóry
+item.hardenedleatherItem.name=Utwardzona Skóra
+
+item.queenbeeItem.name=Królowa pszczół
+item.waxcombItem.name=Plaster Wosku
+item.honeycombItem.name=Plaster Miodu
+item.honeyItem.name=Miód
+item.beeswaxItem.name=Wosk Pszczeli
+item.royaljellyItem.name=Mleczko Pszczele
+item.grubItem.name=Pszczela Larwa
+
+item.grainbaitItem.name=Zbożowa Przynęta
+item.veggiebaitItem.name=Warzywna Przynęta
+item.fruitbaitItem.name=Owocowa Przynęta
+item.fishtrapbaitItem.name=Rybna Przynęta
+
+item.potItem.name=Garnek
+item.skilletItem.name=Patelnia
+item.saucepanItem.name=Rondel
+item.bakewareItem.name=Naczynie Żaroodporne
+item.cuttingboardItem.name=Deska do Krojenia
+item.mortarandpestleItem.name=Moździerz i tłuczek
+item.mixingbowlItem.name=Miska do Mieszania
+item.juicerItem.name=Wyciskarka do Soku
+
+item.asparagusItem.name=Szparag
+item.asparagusseedItem.name=Nasiono Szparagu
+item.grilledasparagusItem.name=Grilowany Szparag
+item.barleyItem.name=Jęczmień
+item.barleyseedItem.name=Nasiono Jęczmienia
+item.beanItem.name=Fasola
+item.beanseedItem.name=Nasiono Fasoli
+item.beetItem.name=Burak
+item.beetseedItem.name=Nasiono Buraka
+item.broccoliItem.name=Brokuł
+item.broccoliseedItem.name=Nasiono Brokuła
+item.cauliflowerItem.name=Kalafior
+item.cauliflowerseedItem.name=Nasiono Kalafiora
+item.celeryItem.name=Seler
+item.celeryseedItem.name=Nasiono Selera
+item.cranberryItem.name=Żurawina
+item.cranberryseedItem.name=Nasiono Żurawiny
+item.garlicItem.name=Czosnek
+item.garlicseedItem.name=Nasiono Czosnku
+item.gingerItem.name=Imbir
+item.gingerseedItem.name=Nasiono Imbiru
+item.leekItem.name=Por
+item.leekseedItem.name=Nasiono Pora
+item.lettuceItem.name=Sałata
+item.lettuceseedItem.name=Nasiono Sałaty
+item.oatsItem.name=Owies
+item.oatsseedItem.name=Nasiono Owsa
+item.onionItem.name=Cebula
+item.onionseedItem.name=Nasiono Cebuli
+item.parsnipItem.name=Pasternak
+item.parsnipseedItem.name=Nasiono Pasternaku
+item.peanutItem.name=Orzech Ziemny
+item.peanutseedItem.name=Nasiono Orzecha Ziemnego
+item.pineappleItem.name=Ananas
+item.pineappleseedItem.name=Nasiono Ananasa
+item.radishItem.name=Rzodkiew
+item.radishseedItem.name=Nasiono Rzodkwi
+item.riceItem.name=Ryż
+item.riceseedItem.name=Nasiono Ryżu
+item.ricecakeItem.name=Ciasto Ryżowe
+item.rutabagaItem.name=Brukiew
+item.rutabagaseedItem.name=Nasiono Brukwii
+item.ryeItem.name=Żyto
+item.ryeseedItem.name=Nasiono Żyta
+item.scallionItem.name=Cebula Dymka
+item.scallionseedItem.name=Nasiono Cebuli Dymki
+item.soybeanItem.name=Soja
+item.soybeanseedItem.name=Nasiono Soi
+item.spiceleafItem.name=Liść Laurowy
+item.spiceleafseedItem.name=Nasiono Liścia Laurowego
```

Comment:
to jest ok, ale tylko dlatego że nie spodziewam się że ktokolwiek zna nazwę tej rośliny :p

Reply:
Szukałem ale nie znalazłem niczego co by lepiej tu pasowało :)